src/mgr/IntervalAction.cc

```diff
@@ -85,6 +86,13 @@ Mgr::IntervalActionData::operator += (const IntervalActionData& stats)
     swap_ins += stats.swap_ins;
     swap_files_cleaned += stats.swap_files_cleaned;
     aborted_requests += stats.aborted_requests;
+    if (Config.paranoid_hit_validation.count()) {
+        hitValidationAttempts += stats.hitValidationAttempts;
+        hitValidationRefusalsDueToLocking += stats.hitValidationRefusalsDueToLocking;
```

Comment:
I wonder whether these increments should be conditional: The condition reflects the _current_ Squid state but this code accumulates historical data, right? The increments are quite cheap and are not on a frequently executed path so we do not have to optimize them away at the expense of accurate stats. If my concerns are valid, please make these increments unconditional, removing the `#include` the condition required.

Reply:
I agree these should be unconditional. The condition should only be checked when doing the actual action+counter.

Reply:
Done.